src/style.css

```diff
@@ -0,0 +1,100 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  color: #060b35;
+}
+
+body {
+  margin-left: 0;
+}
+
+.card_container {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.card_image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card_product-name {
+  margin-top: 40px;
+  margin-bottom: 0;
+
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card_item-number {
+  color: #616070;
+  margin: 5px auto 9px;
+}
+
+.card_product-info {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card_stars {
+  display: flex;
+}
+
+.stars_star {
+  display: inline-block;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars_star:last-child {
+  background-image: url(./images/star.svg);
+}
+
+.card_price-value {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card_price {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card_buy-item-button {
+  display: block;
+  height: 40px;
+
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  line-height: 40px;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.card_buy-item-button:hover {
```

Comment:
But in Figma for hover you should add border: 1px solid #0acdc;

Reply:
yeah, i don't know how i missed a thing that basic haha. nice catch